# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mips/lcdutils/makefile-useprefix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-01-30 19:04:40.000000000 +0100
+++ ./Makefile	2004-01-30 19:04:17.000000000 +0100
@@ -2,12 +2,13 @@
 #LDFLAGS=-static
 
 CFLAGS=-O2 -Wall
+PREFIX=/usr/local
 
 all: putlcd buttond
 
 install: all
-	install putlcd /usr/local/sbin
-	install buttond /usr/local/sbin
+	install putlcd ${PREFIX}/sbin
+	install buttond ${PREFIX}/sbin
 
 clean:
 	rm -f putlcd buttond *.o
